ci/environment-py38.yml

```diff
@@ -0,0 +1,11 @@
+name: xhistogram_test_env
+dependencies:
+  - python=3.8
+  - xarray
+  - dask
+  - numpy
+  - pytest
+  - pip
+  - pip:
+    - codecov
+    - pytest-cov
```

Comment:
This file has no effect unless you also add it to .travis.yml. Anyway, it is superseded by #30. So could you just remove this for now?